recipes/django-leaflet/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "django-leaflet" %}
+{% set version = "0.22.0" %}
+{% set sha256 = "2a1f4f31bff2b3a58158372f7910053670483e3e11f8ac365ce75f3e3a4ac11c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
We can drop the `fn` line for python recipes. I'll modify the example.
(It does sound silly to specify something that is already in the URL.)